math/binary/fast_inverse_sqrt.go

```diff
@@ -0,0 +1,29 @@
+// Calculating the inverse square root
+// [See more](https://en.wikipedia.org/wiki/Fast_inverse_square_root)
+package binary
+
+import (
+	"math"
+)
+
+// Assumes that number always positive
```

Comment:
A minor nit, Go's documenting comments are supposed to be written in a specific way, please refer to the documenting [tips here](https://tip.golang.org/doc/comment).
Eg:

```suggestion
// FastInverseSqrt assumes that number always positive
```